guides/common/attributes-satellite.adoc

```diff
@@ -27,6 +27,7 @@
 :APIDocURL: {BaseURL}api_guide/index#
 :HammerDocURL: {BaseURL}hammer_cli_guide/index#
 :ConfiguringVMSubscriptionsDocURL: {BaseURL}configuring_virtual_machine_subscriptions_in_red_hat_satellite/index#
+:UpdatingDocURL: {BaseURL}updating_red_hat_satellite/index#
```

Comment:
This definition goes under the Upgrading, because it is in the upstream.

Reply:
Addressed :+1:

Reply:
Addressed :+1: